test/fixtures/v5/continue/sharp_left_default.json

```diff
@@ -12,10 +12,13 @@
         "es": "Continua cerrada a la izquierda",
         "fr": "Continuer franchement à gauche",
         "id": "Terus tajam kiri",
+        "it": "Continua Leggera svolta a sinistra",
```

Comment:
I think a sharp left would always be given as a `turn` instruction rather than a `continue` instruction, so it might be safe to ignore the weird output here. Is that correct @bsudekum?